ontology-versions.sparql

```diff
@@ -0,0 +1,6 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+
+CONSTRUCT {
+?ont a owl:Ontology
+?ont owl:versionIRI ?version .
+}
```

Comment:
Is it valid to not have a `WHERE` clause?

Reply:
The updated version doesn't have any shared variables in the CONSTRUCT and the WHERE. You can basically just use the same patterns in each (no need for `?s ?p ?o`). Alternatively, just put your patterns in WHERE, and use the [CONSTRUCT WHERE](https://www.w3.org/TR/sparql11-query/#constructWhere) style.